### ISSUE TITLE
Missing methods

### ISSUE THREAD

**Issue by etranger** [php/doc-en#3934]
From manual page: https://php.net/class.reflectionclassconstant

Two methods from the [RFC](https://wiki.php.net/rfc/typed_class_constants) that are actually supported since [8.3](https://github.com/php/php-src/commit/414f71a90254cc33896bb3ba953f979f743c198c), are still missing in the documentation:

```
class ReflectionClassConstant implements Reflector {
    ...
 
    public function getType(): ?ReflectionType {}
    public function hasType(): bool {}
}
```

**Comment by cmb69** [comment]
Seems to be correct. These new methods are not listed in [UPGRADING](https://github.com/php/php-src/blob/PHP-8.3/UPGRADING) and as such are likely missing in the migration guide (and missing in the [documentation tracker](https://github.com/php/doc-en/issues/2796)).

**Comment by Girgias** [comment]
Fixed by 8660511c7435b20a3ac80641fd124fb3fd04e43e
